.env.app

```diff
@@ -0,0 +1,6 @@
+PORT=9000
```

Comment:
Can't see the reason of renaming .env.example to .env.app
it is just example and not in use
each developer
should create own .env file from .env.example

```sh
cp .env.example .env
```
to override defaults